torchtext/experimental/transforms/transforms.py

```diff
@@ -0,0 +1,60 @@
+from typing import List
+import torch
+
+
+class TokenizerTransform(torch.nn.Module):
+    r""" Initiate a tokenizer transform class
+    """
+
+    def __init__(self, tokenizer, language='en'):
+        super(TokenizerTransform, self).__init__()
+        if tokenizer is None:
+            def split_tokenizer(line: str) -> List[str]:  # noqa: F821
+                re_tokens: List[str] = []
+                for token in line.split():
+                    if token != '':
+                        re_tokens.append(token)
+                return re_tokens
+            self.tokenizer = split_tokenizer
+        elif tokenizer == "basic_english":
+            if language != 'en':
+                raise ValueError("Basic normalization is only available for Enlish(en)")
+
+            def basic_english_normalize(line: str) -> List[str]:
```

Comment:
We have a generic regex functional already I think right?

Reply:
https://github.com/pytorch/text/blob/master/torchtext/data/functional.py#L125

Reply:
Maybe "custom_regex" isn't the best name

Reply:
yes that's true. But there is a `try` block in `re`, which is not jitable. That's why I'm re-writing this part.